code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -432,6 +432,9 @@
 			else
 				message = "moans!"
 				m_type = 2
+				//vorestation edit start
+				playsound(loc, 'sound/voice/moan.ogg', 50, 1, -1)
+				//vorestation edit stop

```

Comment:
No need to do start/stop on a single line vorestation edit.

Reply:
D-does it matter?

Reply:
I prefer for edit markings to be consistent as possible.

Reply:
done